### ISSUE TITLE
Update packaging to support new rexray binaries

### ISSUE THREAD

**Issue by cduchesne** [codedellemc/rexray#702]
Now that REX-Ray can be delivered in multiple forms (combo/client/controller/agent), we should update our default packaging to perhaps include both a client and server, as well as support these new options as standalone installs via our bash install script.

**Comment by koensayr** [comment]
Closing as a duplicate of #666

**Comment by codenrhoden** [comment]
This would want to be done in conjunction with #666

**Comment by codenrhoden** [comment]
@koensayr I'm a fan of keeping this one open. #666 addresses changes needed to the bash script installer, whereas this issue addresses how we package/create our RPM and DEB files. I think this one needs to be done first, and then we use #666 to allow users to consume those packages.
